framework/src/util/SecurityException.cpp

```diff
@@ -0,0 +1,37 @@
+/*=============================================================================
+
+  Library: CppMicroServices
+
+  Copyright (c) The CppMicroServices developers. See the COPYRIGHT
+  file at the top-level directory of this distribution and at
+  https://github.com/CppMicroServices/CppMicroServices/COPYRIGHT .
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+=============================================================================*/
+
+#include "cppmicroservices/SecurityException.h"
+
+namespace cppmicroservices {
+SecurityException::SecurityException(std::string what,
+                                     cppmicroservices::Bundle origin)
+  : std::runtime_error(std::move(what)),
+    origin(std::move(origin))
+{}
+
+Bundle SecurityException::GetBundle() const
+{
+  return origin;
+}
+
+}
```

Comment:
This is pedantic but this file does not have a final newline.

Reply:
I will fix this.